packages/events/src/index.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type {
+  EventBroker,
+  EventParams,
+  EventPublisher,
+  EventSubscriber,
```

Comment:
With the new setup, there is open question whether we still need the publisher and subscriber interfaces.

EventPublisher implementations could receive the `events` instance at the constructor, same for subscribers. There would be no need to expose them at their respective public API.

However, this could be rather a follow-up I guess as changes at implementations of these would make this PR much bigger.